src/server/helpers/middleware.ts

```diff
@@ -291,7 +290,7 @@ export async function validateCollectionParams(req, res, next) {
 		return next(new error.BadRequestError('Invalid collection name: Empty string not allowed', req));
 	}
 	const entityTypes = _.keys(commonUtils.getEntityModels(orm));
-	if (!entityTypes.includes(_.upperFirst(_.camelCase(entityType)))) {
+	if (!entityTypes.includes(.upperFirst(.camelCase(entityType)))) {
```

Comment:
Unrelated change, removing the reference to the lodash library should break the code here.